python/dgl/data/fraud.py

```diff
@@ -213,13 +213,16 @@ class FraudYelpDataset(FraudDataset):
 
     - Nodes: 45,954
     - Edges:
-        R-U-R: 49,315
-        R-T-R: 573,616
-        R-S-R: 3,402,743
-        ALL: 3,846,979
+
+        - R-U-R: 98,630
```

Comment:
What happened here? Did you add reversed edges?

Reply:
> What happened here? Did you add reversed edges?

As the dostring in line 25 described, the original dataset is bidirectional. However, the statistics from original paper hasn't take reversed edges in to account. Therefore, the number of edges should be doubled.